_includes/usagepolicy.html

```diff
@@ -0,0 +1,32 @@
+<div class="full-width-bg component" xmlns="http://www.w3.org/1999/html">
+  <div class="grid-wrapper">
+    <div class="width-6-12 width-12-12-m">
+      <p>This page contains the policy for the Red Hat sponsoered anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
+      <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
+      <H3>What information does Red Hat collect?</H3>
+      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat.  Metrics include the following information such as:</p>
+      <ul>
+        <li>Usage information about components, features and extensions.</li>
+        <li>The number and nature of errors that occur during development.</li>
+        <li>Quarkus version</li>
+        <li>Java version, vendor</li>
+        <li>Build tool used</li>
+        <li>Build duration</li>
+        <li>List of extensions and their versions</li>
+        <li>Timezone</li>
+        <li>CI system name, if available</li>
+        <li>If it's built with kubernetes</li>
+      </ul>
+      <H3>Personal Data</H3>
+      <p>Red Hat does not intend to collect personal information. To the extent that any telemetry data constitutes personal data, please refer to the <a href="https://www.redhat.com/en/about/privacy-policy" target="_blank">Red Hat Privacy Statement</a> for more information about Red Hat’s privacy practices.</p>
+      <H3>Retention</H3>
+      <p>Red Hat retains and stores telemetry data only for as long as it’s needed for the purposes described above or as otherwise required or permitted by law.</p>
+      <H3>Data Security</H3>
+      <p>Red Hat employs technical and organizational measures designed to protect the telemetry data.</p>
+      <H3>Data Sharing</H3>
+      <p>Only personnel with a valid business justification are permitted access to telemetry data. Red Hat may share telemetry data in an aggregated form that does not identify users.</p>
```

Comment:
I thought our data didn't identify users? We need to be consistent in the wording.

Reply:
Yes - but that data is not able to identify users does not mean people with no business justification are permitted to access the resulting data. This is a general data gathering protection requirement IMO.